App_Data/jobs/triggered/status-update/run.sh

```diff
@@ -2,6 +2,6 @@
 
 printf "Invoke status-update endpoint on :%s\n" $PAYMENT_SERVER_URL
 
-curl -X POST $PAYMENT_SERVER_URL/payments/update -d {}
+curl -X PATCH $PAYMENT_SERVER_URL/card-payments/update -d {}
```

Comment:
@eamagdalena - i noticed in the webjob logs. you need one more “*” in cron expression. Azue cron expression is slightly different. 
“Failed to parse schedule “0 30 * * *“. Error: ‘0 30 * * *’ is an invalid crontab expression. It must contain 6 components of a schedule in the sequence of seconds, minutes, hours, days, months, and days of the week.”